Library/Homebrew/cli/named_args.rb

```diff
@@ -38,6 +38,22 @@ def to_formulae_and_casks(only: nil, method: nil)
         end
       end
 
+      def to_formulae_to_casks(method: nil, only: nil)
```

Comment:
What's this method doing? Two arrays one with formulae and one casks?

Reply:
Yeah. I suppose it's possible to do this in `cmd/info`, but given that `to_resolved_formulae_to_casks` is in here already it felt like a good spot